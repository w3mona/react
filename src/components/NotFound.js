import React from 'react';
import {Link} from 'react-router-dom';
const NotFound = () =>(
    <div>
        <h2>404 Not Found</h2>
        <Link to="/" >Go to home</Link>
      </div>
    );


    export default NotFound;