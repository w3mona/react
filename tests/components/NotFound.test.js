import React from 'react';
import {shallow} from 'enzyme';
import NotFound from '../../src/components/NotFound';

test("not found" , ()=>{
   const wrapper = shallow(<NotFound/>);
    expect(wrapper).toMatchSnapshot();
});
