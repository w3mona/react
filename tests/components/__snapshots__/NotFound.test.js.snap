// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found 1`] = `
<div>
  <h2>
    404 Not Found
  </h2>
  <Link
    replace={false}
    to="/"
  >
    Go to home
  </Link>
</div>
`;
